egs/librispeech/asr/simple_v1/mmi_bigram_embeddings_train.py

```diff
@@ -160,14 +162,16 @@ def get_objf(batch: Dict,
             graph_compiler.ctc_topo,
             dense_fsa_vec,
             max_phone_id=graph_compiler.max_phone_id,
-            num_paths=5, # NOTE(fangjun): a larger number results in OOM in `intersect_dense` below
-            debug=True)
+            num_paths=10, # NOTE(fangjun): a larger number results in OOM in `intersect_dense` below
```

Comment:
10 is used here to avoid OOM.

Reply:
@danpovey 

What do you think is an appropriate number here? 

It causes OOM with the modified `intersect_dense` (i.e., the latest k2) if it is set to 25.

Reply:
Mm.  it is not yet as memory efficient as it could be, we need the row-ids function.
I think 10 is more than enough.  For training, 3 to 5 would probably be sufficient.